tests/test_response.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from drf_pdf import response
+import codecs
```

Comment:
With `codecs.open` I can solve the classic unicode problem [1] on py2.7 and py3.4.

[1] https://travis-ci.org/drgarcia1986/drf-pdf/jobs/53496661#L356